Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/inbound/PassthroughInboundDocRetrieve.java

```diff
@@ -43,32 +42,30 @@
 public class PassthroughInboundDocRetrieve extends BaseInboundDocRetrieve {
 
     private final PolicyTransformer pt;
-    private final AuditTransformer at;
     private final InboundDelegate ad;
     private final CONNECTInboundOrchestrator orch;
+    DocRetrieveAuditLogger docRetrieveAuditLogger = new DocRetrieveAuditLogger();
```

Comment:
Two questions:

1. Is default access okay for this, or should it be private?
2. Do we even need to hold this?  Class inherits "getLogger()" from super.